semgrep/semgrep/spacegrep.py

```diff
@@ -45,7 +54,7 @@ def run_spacegrep(
 ) -> dict:
     matches: List[dict] = []
     errors: List[dict] = []
-    targets_time: Dict[str, float] = {}
+    targets_time: Dict[str, Tuple[float, float, float]] = {}
```

Comment:
I just noticed all these kwargs fall prey to mutable default argument gotchas: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
@mschwager didn't @brendongo write a semgrep rule for this?